api/util.py

```diff
@@ -150,6 +168,12 @@ def _update_db(db, datainfo):
             new=True,
             projection=PROJECTION_FIELDS,
             )
+    try:
+        session_label = session['label']
+    except KeyError:  # this can happen if nims_metadata_status == None
+        session_label = '(unknown)'
+    log.info('Using group_id="%s", project_name="%s", and session_label="%s"'
+             % (project['group'], project['name'], session_label))
```

Comment:
Let's try to keep the overall number of log lines per sorted file to a minimum. I'm currently seeing 3 lines per file at ```info``` level, plus one for parsing. One line at ```info``` and one more at ```debug``` seems like a good target. I see the issue of having access to the relevant info at the right time without returning a bunch of basically useless info.

I would play with it myself, but I won't because I can't commit into your PR. If you're going to continue contributing at this rate, it might make sense to make you a commiter for our repos. Thoughts?

Please also try to follow the layout of existing log entires. Note the spacing.

Also, this should be a single line.